### PR TITLE
NDRS-1086: change SSE broadcast channel size to be a factor of event stream buffer size

### PR DESCRIPTION
**Pull request by Fraser999**
Ref: https://casperlabs.atlassian.net/browse/NDRS-1086.

This PR removes `broadcast_channel_size` from the SSE config, instead making the value a factor (120%) of the `event_stream_buffer_length`.

We always want the broadcast channel size to be greater than the event stream buffer length so that a new client can retrieve the entire set of buffered events if desired.

@sacherjj @momipsl note that this PR changes the config.toml.

### CODE REVIEW

File: node/src/components/event_stream_server.rs [hunk 1]
```diff
@@ -77,8 +87,10 @@ impl EventStreamServer {
         let (sse_data_sender, sse_data_receiver) = mpsc::unbounded_channel();
 
         // Event stream channels and filter.
+        let broadcast_channel_size = config.event_stream_buffer_length / 100
```
Comment (goral09) [inline node/src/components/event_stream_server.rs:90]: Is that value in bytes? If so, can we add the suffix `_bytes` to it?

**Review by goral09 — APPROVED** [review]

File: node/src/components/event_stream_server.rs (hunk 1 above)
Comment (Fraser999) [reply node/src/components/event_stream_server.rs:90]: No, they both measure event counts.

**Review by dwerner — APPROVED** [review]

**Comment by Fraser999** [comment]
bors merge

**Comment by Fraser999** [comment]
bors merge

**Pull request merged**